src/app/workflow-list/workflow-list.component.css

```diff
@@ -105,6 +105,7 @@
   display: flex;
   flex: 1 1 auto;
   flex-direction: row;
+  align-items: center;
```

Comment:
The progress indicator should NOT be center aligned, it should be left aligned.

I found that the following produced the correct visual result:
```
@@ -105,7 +105,6 @@
   display: flex;
   flex: 1 1 auto;
   flex-direction: row;
-  align-items: center;
 }
 
 .progress-bar {
@@ -115,6 +114,7 @@
   height: 4px;
   background-color: #CCD2D5;
   border-radius: 2px;
+  justify-content: left;
 }
 
 .progress-bar-filled {
```

Reply:
@rpenny125 ,
Here is what I see with my fix:
![image](https://user-images.githubusercontent.com/34669449/44571983-e7787200-a79b-11e8-9b20-a6c9c15a23e0.png)

I would check this again after we merge latest changes zlux-widgets, zlus-grid, and zlux-workflow into zowe.

Reply:
@rpenny125 Do you still see the issue? I rebased this branch into the master and I don't see any issues with Progress column:
![image](https://user-images.githubusercontent.com/34669449/45338034-a7d7c580-b5a4-11e8-88d2-26fe44369dbf.png)